resources/articles/adaptive_layer_height_variation.md

```diff
@@ -0,0 +1,3 @@
+The maximum deviation from the base layer height (from the selected profile) in either direction.
```

Comment:
To be consistent with the rest of the articles, it would be better to make this a full sentence.